podcasts/Onboarding/Encourage Account Creation/InformationalModalView.swift

```diff
@@ -0,0 +1,87 @@
+import SwiftUI
+
+struct InformationalModalView: View {
+    @EnvironmentObject var theme: Theme
+    @State var currentIndex: Int? = 0
+
+    let viewModel: InformationalModalViewModel
+
+    private let items = InformationalFeatureCardItem.allCases
+    private var isiPad: Bool {
+        UIDevice.current.isiPad()
+    }
+    private var cardHeight: CGFloat {
+        isiPad ? 274 : 370
```

Comment:
Instead of checking for iPad did you consider checking for size class?
Like this:
@Environment(\.horizontalSizeClass) var horizontalSizeClass

This could be relevant for iPad multitask modes